src/proto/mod.rs

```diff
@@ -199,6 +202,7 @@ impl<S: Read + Write> Client<S> {
                 return Err(ConnectError::AuthenticationNeeded.into());
             }
         }
+
```

Comment:
Was this intentional?

Reply:
Yes, I thought having a blank line here like other blocks in this function/crate made sense. Happy to change it back if you prefer.